pom.xml

```diff
@@ -684,6 +684,7 @@
           <configuration>
             <configLocation>beam/checkstyle.xml</configLocation>
             <headerLocation>beam/header-file.txt</headerLocation>
+	    <suppressionsLocation>beam/suppressions.xml</suppressionsLocation>
```

Comment:
Incidentally, all of these would be better as `org/apache/beam/...` since they live in a global namespace.